zio-ftp/src/main/scala/zio/ftp/UnsecureFtp.scala

```diff
@@ -102,6 +102,11 @@ final private class UnsecureFtp(unsafeClient: Client) extends FtpAccessors[Clien
 object UnsecureFtp {
   type Client = JFTPClient
 
+  def apply(unsafeClient: Client): URIO[Scope, FtpAccessors[Client]] =
+    acquireRelease(ZIO.succeed(new UnsecureFtp(unsafeClient))) { client =>
+      client.execute(_.logout()).ignore *> client.execute(_.disconnect()).ignore
+    }
```

Comment:
This is solved the issue, but I think it makes quite a confusing API with apply of the client, which assumes it will be already connected.

I would approach it differently and provide a builder via `UnsecureFtpSettings` 
```
```

Reply:
thanks for your tip. i made some changes.

I dont see how we could provide the same implementation.

Reply:
I'll try to submit a proposal later this week, this impl might still have issues as some settings needs to be apply after connecting or login in.